### ISSUE TITLE
AWT integration tests: Native: False positives on systems with older libs (fontconfig ?)

### ISSUE THREAD

**Issue by Karm** [quarkusio/quarkus#22652]
### Describe the bug

I came across JDK x Mandrel portable builds (regardless of Mandrel version, both 21.2, 21.3 and 22.0) that pass Quarkus AWT tests on RHEL 9 :heavy_check_mark: , but fail on RHEL 8 :x: :
```
TIFF: Wrong pixel at 275x199. Expected: [255,0,0,255] Actual: [171,37,162,255] ==> expected: <true> but was: <false>
PNG:  Wrong pixel at 275x199. Expected: [255,0,0,255] Actual: [171,37,162,255] ==> expected: <true> but was: <false>
JPG:  Wrong pixel at 275x199. Expected: [254,2,0,0]   Actual: [174,36,157,0]   ==> expected: <true> but was: <false>
BMP:  Wrong pixel at 275x199. Expected: [255,0,0,0]   Actual: [171,37,162,0]   ==> expected: <true> but was: <false>
```
The test was looking for an edge of a character rendered with a specific font.

This might come down to a different native library version being used (fonts?) and needs attention.

I will either attend to the issue on the *Mandrel* side or adjust the *AWT test suite* to be more forgiving than it already is with thresholding expected pixel values.

**Comment by Karm** [comment]
![difference_animation](https://user-images.githubusercontent.com/691097/148514301-df34f569-1563-4d72-854e-f0f83b21fa01.gif)

The animation shows the difference. The two top left texts are rendered with font embedded in the test suite whereas the rest of the text is rendered using default fonts either brought with the JDK or present in the system. I cannot reproduce it on a stock, clean install of RHEL 8.5. It looks the same as it does on modern Fedora, Centos 8 Stream, Ubuntu (including GitHub Actions instances). It is just a particular RHEL 8 setup that renders a different image. I will dig deeper and modify the TS to run everywhere as this is a false positive.

@zakkak FYI to see the nice picture :-D

**Comment by Karm** [comment]
The root cause was different system fonts, not libraries. Solved in #22755
